src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md

```diff
@@ -36,6 +36,8 @@ Get-PowerBIWorkspace [-Scope <PowerBIUserScope>] [-Filter <String>] [-User <Stri
 
 ## DESCRIPTION
 Retrieves a list of Power BI workspaces that match the specified search criteria and scope.
+By default (without [-First](https://github.com/Microsoft/powerbi-powershell/blob/master/src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md#-first)) it returns the first 100 workspaces.
```

Comment:
Say: The first 100 workspaces are returned, use -First and -Skip to fetch more workspaces or use -All to return all workspaces